### PR TITLE
Skeleton tabnav to put forms in

### PR DESCRIPTION
**Pull request by bamader**
Assuming the TabNav PR in the component library gets approved, here's the blank field form navigation from the edit roster page. UI is a work in progress but this at least lets us start getting forms created. Also, as we make them, we can refactor the currently hardcode TabNavProps objects in the middle of CheckData to be their own module entities.

Screen Shot 2020-08-20 at 12 21 03 PM

### CODE REVIEW

File: client/src/containers/CheckData/CheckData.tsx [hunk 1]
```diff
@@ -106,28 +107,58 @@ const CheckData: React.FC = () => {
...
-                        type="input"
-                        onChange={() => {}}
-                      />
-                      <Button
-                        className="margin-top-2"
-                        text="A button can go here"
-                      />
-                    </div>
-                    <div className="grid-col-1">
+                    <div >
```
Comment (jlhogan) [inline client/src/containers/CheckData/CheckData.tsx:113]: we went back and forth on this, but actually want to update this to be a separate page, not a row expansion. So the name becomes a button that links to a new route, which shows a page containing the tab nav and some header content (child's name etc)

Comment (bamader) [reply client/src/containers/CheckData/CheckData.tsx:113]: Ah gotcha. Will update.

Comment (bamader) [reply client/src/containers/CheckData/CheckData.tsx]: How's this?

Screen Shot 2020-08-20 at 1 56 03 PM

File: client/src/containers/CheckData/CheckData.tsx [hunk 2]
```diff
@@ -15,8 +15,10 @@ import {
   Column,
   TextInput,
   ExpandRow,
+  TabNav,
 } from '@ctoec/component-library';
 import { ReactComponent as Arrow } from '@ctoec/component-library/dist/assets/images/arrowRight.svg';
+import { ProcessStep } from '@ctoec/component-library/dist/components/ProcessList/ProcessStep';
```
Comment (jlhogan) [inline client/src/containers/CheckData/CheckData.tsx:21]: nit: unused import

**Pull request merged**